snmp/datadog_checks/snmp/data/auto_conf.yaml

```diff
@@ -73,3 +73,9 @@ instances:
     ## Name of your context (optional SNMP v3-only parameter).
     #
     context_name: "%%extra_context_name%%"
+
+    ## @param autodiscovery_subnet - string - optional
+    ## The autodiscovery subnet the device is part of.
+    ## Used by Agent autodiscovery to pass subnet name.
+    #
+    autodiscovery_subnet: "%%autodiscovery_subnet%%"
```

Comment:
What about not having a new parameter, but setting `tags` directly?

Reply:
@therve Good point. Updated to use tags instead.